homeassistant/components/flo/sensor.py

```diff
@@ -109,29 +128,55 @@ def unit_of_measurement(self) -> str:
 class FloTemperatureSensor(FloEntity):
     """Monitors the temperature."""
 
-    def __init__(self, device):
+    def __init__(self, name, device):
         """Initialize the temperature sensor."""
-        super().__init__("temperature", NAME_TEMPERATURE, device)
+        super().__init__("temperature", name, device)
         self._state: float = None
 
     @property
     def state(self) -> Optional[float]:
         """Return the current temperature."""
         if self._device.temperature is None:
             return None
-        return round(fahrenheit_to_celsius(self._device.temperature), 1)
+        return round(self._device.temperature, 1)
```

Comment:
so I think I did this originally because HA will auto adjust it based on what the user configures. I need to go through the old PR's to see if this was something I was asked to do.

Reply:
nope... this is something I should have done myself... :( https://github.com/home-assistant/core/pull/38171#discussion_r467460761